src/pages/lib/ssr/nextjs/q/platform/[platform].mdx

```diff
@@ -118,7 +124,7 @@ export default function ConfigureAmplifyClientSide() {
 ```
 
 `layout.tsx`:
-```typescript
+```jsx
```

Comment:
does `tsx` not work?

Reply:
`tsx` doesn't apply any syntax highlighting :/